lib/fastlane/plugin/appcenter/actions/appcenter_codepush_release_react.rb

```diff
@@ -0,0 +1,156 @@
+require 'fastlane_core/ui/ui'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+
+  module Actions
+    class AppcenterCodepushReleaseReactAction < Action
+      def self.description
+        "CodePush release react action"
+      end
+
+      def self.authors
+        ["Ivan Sokolovskii"]
+      end
+
+      def self.run(params)
+        app = params[:app_name]
+        owner = params[:owner_name]
+        token = params[:api_token]
+        deployment = params[:deployment]
+        dev = paramss[:development]
+        description = params[:description]
+        mandatory = params[:mandatory]
+        version = params[:target_version]
+        disabled = params[:disabled]
+        no_errors = params[:no_duplicate_release_error]
+        bundle = params[:bundle_name]
+        output = params[:output_dir]
+        sourcemap_output = params[:sourcemap_output]
+        dry_run = params[:dry_run]
+
+        command = "appcenter codepush release-react --token #{token} --app #{owner}/#{app} --deployment-name #{deployment} --development #{dev} "
+        if description
+          command += "--description \"#{description}\" "
+        end
+        if mandatory
+          command += "--mandatory "
+        end
+        if version
+          command += "--target-binary-version #{version} "
+        end
+        if disabled
+          command += "--disabled "
+        end
+        if no_errors
+          command += "--disable-duplicate-release-error "
+        end
+        if bundle
+          command += "--bundle-name #{bundle} "
+        end
+        if output
+          command += "--output-dir #{output} "
+        end
+        if sourcemap_output
+          command += "--sourcemap-output #{sourcemap_output} "
+        end
+        if dry_run
+          UI.message("Dry run!".red + " Would have run: " + command + "\n")
+        else
+          sh(command.to_s)
+        end
```

Comment:
```suggestion
          end
        end
```

Reply:
@iv0rish What is going on here? Why add additional indentation? It's an end for `else` clause on line 59.

Reply:
This is typo. Sorry